web_poet/page_inputs.py

```diff
@@ -123,6 +145,27 @@ def declared_encoding(self) -> Optional[str]:
         return http_content_type_encoding(content_type)
 
 
+Mapping = Dict[str, str]
```

Comment:
a more specific name might be better, as a reader might confuse it with https://docs.python.org/3/library/collections.abc.html#collections.abc.Mapping when reading function definitions

Reply:
Good point! Renamed this into `_StrMapping` in https://github.com/scrapinghub/web-poet/pull/22/commits/9f43187c5944c0cf0ce3bc122f3c00580262dec1.